public/index.html

```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
     </div>
   </div>
 </div>
+<!-- HCA Application End -->
 
+<!-- Maintenance Page Start -->
+  <!--<div class="splash">
```

Comment:
We're keeping this commented out so we can uncomment it and deploy if anything goes wrong?